src/scripts/review.bats

```diff
@@ -221,3 +221,42 @@ setup() {
 
 	done
 }
+
+@test "RC011: Ensure usage examples showcase current major version of the orb." {
+	if [[ "${SKIPPED_REVIEW_CHECKS[*]}" =~ "RC001" ]]; then
+		skip
+	fi
+
+	if [[ -z "$CIRCLE_TAG" ]]; then
+		echo "No tag detected. Skipping usage example check."
+		skip
+	fi
+
+	if [[ ! "$CIRCLE_TAG" =~ ^v[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+		echo "Non-production tag detected. Skipping usage example check."
+		skip
+	fi
+
+	CURRENT_MAJOR_VERSION=$(echo "$CIRCLE_TAG" | cut -d '.' -f 1)
+
+	for i in $(find "${ORB_SOURCE_DIR}examples/*.yml" -type f); do
+		ORB_REF_STRING=$(yq ".usage.orbs[\"${ORB_VAL_ORB_NAME}\"]" "$i")
+		ORB_REF_VERSION_STRING=$(echo "$ORB_REF_STRING" | cut -d '@' -f 2)
+		ORB_REF_MAJOR_VERSION=$(echo "$ORB_REF_VERSION_STRING" | cut -d '.' -f 1)
+
+		if [[ "$ORB_REF_MAJOR_VERSION" != "$CURRENT_MAJOR_VERSION" ]]; then
+			echo "File: \"${i}\""
+			echo "Orb version: \"${ORB_REF_VERSION_STRING}\""
```

Comment:
The term `Orb version` is a bit misleading here. I thought it would show me the current version of my orb, but it shows the version in the examples. We should re-word it so it's more clear it's showing the example's version. Maybe `Orb version in the example: ` or something along these lines.